app/javascript/app/components/card/card-row/card-row-styles.scss

```diff
@@ -1,49 +1,30 @@
 @import '~styles/settings.scss';
 
-$standard-text-padding: 20px;
-$extra-text-padding: 30px;
-
 .cardRow {
-  background-color: $white;
-  border-top: 1px solid $gray3;
+  background-color: white;
```

Comment:
Why we don't use the $white variable? I know its a bit verbose but I prefer not to have hardcoded colors

Reply:
Thanks, you're right, I forgot to set it that way when I was messing with card components, will change it ;)